collectors/tidpost

```diff
@@ -40,21 +41,20 @@ class TidmarshReceiver:
         return self.subscriber.recv()
 
 
-# the sensors don't self-describe their units, so we map them here. Note that
-# the charge_flags are unpacked
+# the sensors don't self-describe their units, so we map them here. (Legacy
+# protocol only; this is now deprecated as in v2 sensors *do* self-describe
+# their units)
 TIDMARSH_UNITS = {
     "sht_humidity": "percent",
     "bmp_pressure": "hPa",
     "battery_voltage": "volts",
     "illuminance": "lux",
-    "sht_temperature": "celsius",
-    "bmp_temperature": "celsius",
-    "charge_flags_fault": "boolean",
-    "charge_flags_charge": "boolean",
-    "accel_x": "m/s^2",
-    "accel_y": "m/s^2",
-    "accel_z": "m/s^2",
-    "analog": "ADC units"
+    "sht_temperature": "°C",
+    "bmp_temperature": "°C",
+    "accel_x": "g",
+    "accel_y": "g",
+    "accel_z": "g",
+    "analog": "LSB"
 }
 
```

Comment:
Just want to make sure you know - Units are stored on a per-sensor basis (not per-measurement), so renaming these units will only effect newly-created sensors. If data gets posted to sensors that already exist but had different units, the new data value will get posted but the sensor will still have the old unit.